### PR TITLE
mv: refresh page

### PR DESCRIPTION
**Pull request by diraneyya**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
Note: All checks run on the `main` branch only, we are planning to remove the master soon. So I have changed this PRs base branch to `main`.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/mv.md [hunk 1]
```diff
@@ -3,11 +3,15 @@
 > Move or rename files and directories.
 > More information: <https://www.gnu.org/software/coreutils/mv>.
 
-- Move a file to an arbitrary location:
+- Rename a file or an empty directory (`{{target}}` does not exists):
```
[the remark this answers is not in the document]
Comment (diraneyya) [reply pages/common/mv.md:6]: Yes! thank you @kbdharun I actually made the correction in class live in front of my students and was not very focused. You are right, directories can also be renamed even when non-empty (silly me!).

**Review by kbdharun — APPROVED** [review]

**Comment by blueskyson** [comment]
Why should this be split into two examples? I mean "move source to arbitrary location" already includes both situations where the target path exists or not.

**Comment by diraneyya** [comment]
In the source code of the `mv` command [here](https://github.com/MaiZure/coreutils-8.3/blob/master/src/mv.c) you can see in the first line the comment `/* mv -- move or rename files`, indicating that moving things and renaming them are not equivalent, from a logical standpoint.

In the code, the function `target_directory_operand` returns a bool value to indicate whether the last operand is a directory or not, which is needed to decide if a move is possible, versus deciding in favour of a renaming operation.

The fact whether the last operand is a directory or not, plays a big role in determining what is allowed and what is not. For example, if the last operand is a directory, then you can supply multiple files to the `mv` command, all of which will be moved to that directory, supplied last.

If the last operand, on the other hand, was not a directory, then the operation is interpreted as a renaming operation in which only one other operand can be supplied to the `mv`command. Since multiple files cannot be renamed using `mv`.

Trying to supply more than two operands to `mv` implies a moving operation, rather than a renaming operation. For moves to work, the last operand **should be a directory**, not supplying a directory will trigger the error `<operand> is not a directory` and the command will fail.

If moving and renaming were logically equivalent operations, then they can accept the same number of operands. But they do not.

Another evidence for the distinction between moving and renaming is the discrepancy between the following:
```
mv {{existing-file}} {{another-existing-file}}
mv {{existing-folder}} {{another-existing-folder}}
```

If the two operations were logically equivalent, then both above should move or copy. In reality, the first renames a file (deleting the one that existed with the same name before that) while the second moves a directory to another directory.

In contrast:
```
mv {{existing-file}} {{non-existing-file}}
mv {{existing-folder}} {{non-existing-folder}}
```

Renames both the file and the directory.

The dual role of many Linux commands such as this one and the duality of `touch` as either creating an empty file or updating the modification date, is only well understood by those who were exposed to these commands early in their development. New comers to Linux and coding, on the other hand, deserve to have this dual behaviour highlighted and explained to them.

The fact that TLDR pages are an educational resource especially for the uninitiated makes it even more important that these pages are written with distinct logical meanings in mind.

File: pages/common/mv.md [hunk 2]
```diff
@@ -3,13 +3,17 @@
 > Move or rename files and directories.
 > More information: <https://www.gnu.org/software/coreutils/mv>.
 
-- Move a file to an arbitrary location:
+- Rename a file or directory when the target is not an existing directory:
 
 `mv {{source}} {{target}}`
```
Comment (blueskyson) [inline pages/common/mv.md:8]:
```suggestion
- Rename a file or directory when the target is not an existing file or directory:

`mv {{source}} {{target}}`
```

File: pages/common/mv.md [hunk 3]
```diff
@@ -3,13 +3,17 @@
...
 
-- Move files into another directory, keeping the filenames:
+- Move a single file when the target is an existing directory:
 
-`mv {{source1}} {{source2}} {{source3}} {{target_directory}}`
+`mv {{source}} {{target}}`
+
+- Move multiple files into an existing directory, keeping the filenames unchanged:
+
+`mv {{source1}} {{source2}} {{source3}} {{target}}`
```
Comment (blueskyson) [inline pages/common/mv.md:16]:
```suggestion
`mv {{source1}} {{source2}} {{source3}} {{existing_directory}}`
```

File: pages/common/mv.md [hunk 4]
```diff
@@ -3,13 +3,17 @@
 > Move or rename files and directories.
 > More information: <https://www.gnu.org/software/coreutils/mv>.
 
-- Move a file to an arbitrary location:
+- Rename a file or directory when the target is not an existing directory:
 
 `mv {{source}} {{target}}`
 
-- Move files into another directory, keeping the filenames:
+- Move a single file when the target is an existing directory:
```
Comment (blueskyson) [inline pages/common/mv.md:10]:
```suggestion
- Move a file or directory into an existing directory:
```

File: pages/common/mv.md [hunk 5]
```diff
@@ -3,13 +3,17 @@
...
 
-`mv {{source1}} {{source2}} {{source3}} {{target_directory}}`
+`mv {{source}} {{target}}`
```
Comment (blueskyson) [inline pages/common/mv.md:12]:
```suggestion
`mv {{source}} {{existing_directory}}`
```

**Review by blueskyson — COMMENTED** [review]
@diraneyya Okay, I understand your concern. `mv` performs renaming or moving depending on the existence of the target path. Will it make more sense after these suggestions?

File: pages/common/mv.md (hunk 2 above)
Comment (diraneyya) [reply pages/common/mv.md:8]: No this is incorrect. Just keep it as before. Saying "if it is not an existing directory".

The thing is, if it is an existing file and you are renaming a file, then the existing file will be rewritten. If you are renaming a folder, on the other hand, then the `mv` command will come back with an error saying that `{{target}} is not a directory`

**Comment by diraneyya** [comment]
I think this is good enough for now. At least it addresses this very basic command in a logical way in the most common use cases. Thanks to everyone who was involved and I am looking forward to this getting approved and merged.

**Review by blueskyson — APPROVED** [review]
Look good to me. Thanks for contribution.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**